### PR TITLE
Popover / tooltip windows

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
Those changes add wrapper components for tooltips / popovers. I'm still struggling with dynamic loading (see part of the conversation on the topic in https://github.com/angular/angular/issues/8941) but we will need those wrappers anyway. 

Hopefully we will start connecting all those elements together soonish :-)

### CODE REVIEW

File: src/popover/popover.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {Component, Input, ChangeDetectionStrategy} from '@angular/core';
+
+@Component({
+  selector: 'ngb-popover-window',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {'[class]': '"popover " + placement', 'role': 'tooltip'},
```
Comment (Foxandxss) [inline src/popover/popover.ts]: Wouldn't this override all classes used in the component?

Comment (pkozlowski-opensource) [inline src/popover/popover.ts]: huh? This is just a popover decorator, a component that we would be opening ourselves (user's content go into ng-content) so I'm not sure what you mean...

Comment (Foxandxss) [inline src/popover/popover.ts]: Oh right, I meant that if you do `<ngb-popover-window class="foo bar baz"...` those 3 classes would be removed to add the popover-placement one.

Comment (pkozlowski-opensource) [inline src/popover/popover.ts]: Correct. It is just that we are never going to do this in practice.

Comment (Foxandxss) [inline src/popover/popover.ts]: 👍

**Pull request closed without merging**